src/python/WMCore/MicroService/Unified/RequestInfo.py

```diff
@@ -461,8 +461,10 @@ def unified(svc, requestRecords, logger):
             ncopies = getNCopies(cput)
 
             attrs = winfo[wname]
-            ndatasets = len(attrs['datasets'])
-            npileups = len(attrs['pileups'])
+            campaign = attrs['campaign']
+            campaignType = 'primary' # TODO: how to define it?
```

Comment:
That's a good question. I believe it's defined in this method:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/MicroService/Unified/MSTransferor.py#L115

when it's looking for all possible data that we need to transfer (it misses the parent though). So we should not hardcode it but actually see all possible values and get it.